pkg/ocm/clusters.go

```diff
@@ -409,35 +377,6 @@ func (c *Client) GetSubscriptionBySubscriptionID(id string) (*amv1.Subscription,
 
 // GetCluster gets a cluster key that can be either 'id', 'name' or 'external_id'
 func (c *Client) GetCluster(clusterKey string, creator *aws.Creator) (*cmv1.Cluster, error) {
```

Comment:
@machi1990 Should we just drop this function and make `getCluster` exported? I can't really see any benefit any more of having this function call directly another without doing anything else.

Reply:
Yes.

Reply:
done